helm/aws-load-balancer-controller/values.yaml

```diff
@@ -87,6 +87,11 @@ enableCertManager: false
 # ingresses without ingress class annotation and ingresses of type alb
 ingressClass: alb
 
+# The ingressClassParams the ingress class refers to and enforce settings for a set of Ingresses when using with ingress Controller.
```

Comment:
nit: please use the following format for comments
# ingressClassParams specifies the IngressClassParams ...